File: mean.js
const input = require("./input");

(async() => {
    
    const numbers = await input("Enter numbers (Separated by Comma): ");
    const num_list = numbers.replace(" ", "").split(",");
    const num_length = num_list.length;

    let sum = 0;

    // Total sum of all numbers
    num_list.forEach(number => sum += Number(number));

    // Get the average
    const result = sum / num_length;

    console.log(`The mean of ${num_list.join(", ")} is ${result}`);

})();

File: mean.py
numbers = input("Enter integers (Separated by Comma): ")
num_list = numbers.replace(" ", "").split(",")
num_length = len(num_list)

sum = 0

# Sum of all inputs
for item in num_list:
    sum += int(item)

# Get the average
result = sum / num_length

print(f"The mean of {", ".join(num_list)} is {result}")